tests/plugins/test_duplicated_script_tags.py

```diff
@@ -97,3 +97,33 @@ def test_excluded_tag(self):
         results = list(plugin.run())
 
         self.assertEqual(len(results), 0)
+
+    def test_excluded_dependencies(self):
+        path = Path("gsf/PCIDSS/v2.0/PCI-DSS-2.0.nasl")
+        content = (
+            'script_dependencies(name:"Test", type:"checkbox");\n'
```

Comment:
The syntax of `script_dependencies()` is basically `script_dependencies("vt1.nasl", "vt2.nasl");`